packages/app/src/components/search/SearchModal.tsx

```diff
@@ -93,11 +84,6 @@ export const SearchModal = ({ toggleModal }: { toggleModal: () => void }) => {
     searchBarRef?.current?.focus();
   });
 
-  const handleSearchResulClick = useCallback(() => {
-    toggleModal();
-    setTimeout(focusContent, transitions.duration.leavingScreen);
```

Comment:
I found an unexpected behaviour with this - since this was used on `onKeyDown` https://github.com/backstage/backstage/pull/15873/files#diff-506348f267c82faeef8233367f and the key was never checked, when i started to tab through the search modal, it after a couple of second just closed down.